hack/verify-govet.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+run_individual_tests() {
+  files_to_test=$1
+  tests="printf methods structtags composites asmdecl assign atomic bool buildtags copylocks nilfunc rangeloops unreachable shadow unsafeptr unusedresult"
+
+  for test in $tests
+  do
+    touch "_output/govet/${test}.log"
+    for file in $files_to_test
+    do
+      go tool vet "-${test}=true" $file 2>&1 | sed '/exit status/d' >> "_output/govet/${test}.log" || true
+    done
+    if [ -s "_output/govet/${test}.log" ]
+    then
+      echo "FAILURE: go vet -${test} had errors: "
```

Comment:
for now, run with `-all=true -composites=false -printf=false -copylocks=false` so we can get the main checks in (there are still lots of things flagged with those turned on), then we can follow up with sweeps to fix and re-enable those checks